suites/pacific/rados/tier-2_rados_test-pool-functionalities.yaml

```diff
@@ -255,16 +244,18 @@ tests:
               disable_pg_autoscale: true
       desc: many clients clients accessing same PG with bluestore as backend
 
-  - test:
-      name: Migrate data bw pools.
-      module: test_data_migration_bw_pools.py
-      polarion-id: CEPH-83574768
-      config:
-        pool-1-type: replicated
-        pool-2-type: replicated
-        pool-1-conf: sample-pool-1
-        pool-2-conf: sample-pool-2
-      desc: Migrating data between different pools. Scenario-1. RE -> RE
+# Following test has been commented till RCA is complete for massive increase
+# in test completion time
+#  - test:
+#      name: Migrate data bw pools.
+#      module: test_data_migration_bw_pools.py
+#      polarion-id: CEPH-83574768
+#      config:
+#        pool-1-type: replicated
+#        pool-2-type: replicated
+#        pool-1-conf: sample-pool-1
+#        pool-2-conf: sample-pool-2
+#      desc: Migrating data between different pools. Scenario-1. RE -> RE
 
   - test:
```

Comment:
Even this can be commented. The tests are passing. But it's slow. Let's add all tests post RCA itself.

Reply:
All the tests are passing, the other 3 are slow, this one isn't slow, so was not removed. However, will comment it out as discussed

Reply:
Correction: This test is also slow now
`Migrate data bw pools.           Migrating data between different pools. Scenario-2. RE -> EC   2:10:02.742211                   Pass`
Disabling it for now